src/components/toastNotificationsList/toastNotificationsList.js

```diff
@@ -21,46 +25,51 @@ class ToastNotificationsList extends React.Component {
   };
 
   render() {
-    const { toasts, paused } = this.props;
+    const { toasts, timeout } = this.props;
 
     return (
-      <ToastNotificationList>
-        {toasts &&
-          toasts.map((toast, index) => {
-            if (!toast.removed) {
-              return (
-                <TimedToastNotification
-                  key={helpers.generateId('key')}
-                  toastIndex={index}
-                  type={toast.alertType}
-                  paused={paused}
-                  onDismiss={() => this.onDismiss(toast)}
-                  onMouseEnter={this.onHover}
-                  onMouseLeave={this.onLeave}
-                >
-                  <span>
-                    <strong>{toast.header}</strong> &nbsp;
-                    {toast.message}
-                  </span>
-                </TimedToastNotification>
-              );
-            }
-
-            return null;
-          })}
-      </ToastNotificationList>
+      <AlertGroup isToast className="quipucords-toast-notifications__alert-group">
```

Comment:
required:
can't seem to find a role on the source of AlertGroup, but we do see `aria-atomic` being set on AlertGroup related to the use of `isLiveRegion`. confirm with accessibility we don't need to drop a role attribute on AlertGroup, or the `isLiveRegion` on AlertGroup too

Reply:
We have two PF4 `AlertGroup` demos that demo the use of `isLiveRegion` ! Though neither of them use a `role` attribute on `AlertGroup`, they do use an `aria-label` on `Alert`... should we add those @cdcabrera ?

Reply:
per our accessibility team..."[using the AlertGroup `isLiveRegion` with `isToast` is preferable when the region/component is maintained in a page. the `isLiveRegion` for the Alert component is not as reliable with certain types of screen-readers, and it is not recommended.]" ... this is one of those the docs need to be updated on pf so checking with the UXD accessibility team is the path

Reply:
Ah, I see. Now I realize since we have a specific use-case where we are using the toast group (no PF4 react demo for that). Checking with accessibility in those situations is a good thing to keep in mind - will definitely plan to do that moving forward 👍